corehq/apps/case_search/utils.py

```diff
@@ -343,22 +393,22 @@ def get_child_case_types(app, case_type):
 
     Returns a set of case types
     """
-    case_types = set()
+    child_case_types = set()
     for module in app.get_modules():
         if module.case_type == case_type and module_offers_search(module):
             for tab in module.search_detail("long").tabs:
                 if tab.has_nodeset and tab.nodeset_case_type:
-                    case_types.add(tab.nodeset_case_type)
+                    child_case_types.add(tab.nodeset_case_type)
 
-    return case_types
+    return child_case_types
 
 
-def get_child_case_results(helper, parent_cases, case_types):
-    parent_case_ids = {c.case_id for c in parent_cases}
-    results = (helper.get_base_queryset()
-               .case_type(case_types)
-               .get_child_cases(parent_case_ids, "parent")
-               .run().hits)
+def get_child_case_results(helper, parent_case_ids, child_case_types=None):
+    filter = (helper.get_base_queryset().get_child_cases(parent_case_ids, "parent"))
```

Comment:
nit: redundant parenthesis 
```suggestion
    filter = helper.get_base_queryset().get_child_cases(parent_case_ids, "parent")
```

Reply:
[26e3ffa](https://github.com/dimagi/commcare-hq/pull/32587/commits/26e3ffaa29ec122db06e7e0f56f7d75381d7d213)